src/inc/msquic.h

```diff
@@ -704,6 +704,7 @@ void
 #ifdef QUIC_API_ENABLE_PREVIEW_FEATURES
 #define QUIC_PARAM_GLOBAL_VERSION_SETTINGS              0x01000007  // QUIC_VERSION_SETTINGS
 #define QUIC_PARAM_GLOBAL_LIBRARY_GIT_HASH              0x01000008  // char[64]
+#define QUIC_PARAM_GLOBAL_RAW_DATAPATH_PROCS            0x01000009  // uint32_t[]
```

Comment:
I believe we use `uint16_t` everywhere internally for CPU index. I think that's plenty. Not sure we need to expose a uint32_t for this. @thhous-msft thoughts?

Reply:
Currently yes it is a uint16_t for everything, so we don't need to expose it as a uint32_t (Yet)